common/flatpak-dir.c

```diff
@@ -14198,7 +14198,7 @@ flatpak_dir_find_remote_related_for_metadata (FlatpakDir         *self,
                                    state->remote_name,
                                    &url,
                                    error))
-    return FALSE;
+    return NULL;
```

Comment:
I agree that this should return `NULL` rather than `FALSE`, but why does `ostree_repo_remote_get_url` error out at all if installing a flatpak bundle is a supported use case? Why do we not hit the `if (*url == 0)` path below? Is the URL ' ' rather than ''?

Reply:
Also, while NULL is better than FALSE both have the value 0, so this is not really a change in behaviour.

Reply:
I think it was a confusion in commit message. I never said(or intended to) that `ostree_repo_remote_get_url` errors out. It doesn't error out.